services/thumbnails/pkg/config/grpc.go

```diff
@@ -1,7 +1,10 @@
 package config
 
-// GRPC defines the available grpc configuration.
-type GRPC struct {
-	Addr      string `yaml:"addr" env:"THUMBNAILS_GRPC_ADDR" desc:"The address off the grpc service."`
-	Namespace string `yaml:"-"`
+import "github.com/owncloud/ocis/v2/ocis-pkg/shared"
+
+// GRPCConfig defines the available grpc configuration.
+type GRPCConfig struct {
+	Addr      string                 `yaml:"addr" env:"THUMBNAILS_GRPC_ADDR" desc:"The address off the grpc service."`
```

Comment:
```suggestion
	Addr      string                 `yaml:"addr" env:"THUMBNAILS_GRPC_ADDR" desc:"The address of the grpc service."`
```
Beside the typo fixed, I am asking myself what is meant with address and what to enter...?

Reply:
I really don't think that this is the place in the code where we should explain what a bind address is. If we really need to explain it in our docs I suggest to do that in another more generic place (instead of repeating it in the roughly 70 places where we currently allow to set such a parameter)

"The bind address specifies the IP Address (or hostname) and Port that the service should bind to/listen on." Still I would assume, that every admin that needs to set any of these parameters as already heard the term "bind address" before.